vm/runtime.ts

```diff
@@ -1467,12 +1482,8 @@ module J2ME {
       }
     }
 
-    if (false && methodTimeline) {
-      fn = profilingWrapper(fn, methodInfo, methodType);
-    }
-
-    if (traceWriter) {
-      fn = tracingWrapper(fn, methodInfo, methodType);
+    if (profile) {
+      fn = wrapMethod(fn, methodInfo, methodType);
```

Comment:
I was contemplating ripping that out all together because the same thing can be achieved with PROFILE=2 and it is more accurate now since it shows all method enters/exits. I'll restore for now, but something to think about.